modules/core/src/utils/iterable-utils.js

```diff
@@ -26,7 +26,7 @@ const placeholderArray = [];
  * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols
  * and a "context" tracker from the given data
  */
-export function createIterable(data) {
+export function createIterable(data, startIndex = 0, endIndex) {
```

Comment:
For clarify maybe add a default value to the last param as well `endIndex = Infinty` or `null`...